doc/ARCHITECTURE.md

```diff
@@ -172,3 +185,18 @@ The following metrics are collected from the webhook events
         hasDocChapters: boolean;
       }
     ```
+
+- Documentation Updated Metrics
+
+  - Event used - pull-request (closed and merged)
```

Comment:
should this check:
https://github.com/deven-org/telemetry-functions/pull/102/files#diff-4d52b2c0eaa2e938ad2c923c877385443b689d834ec8c089b4e8418ba41adeb9R18
not make sure that the metric is only be called with a merged PR and not every closed one?

Reply:
Yes, exactly. That's why the check returns the boolean of the `merged` property. So it only applies if the PR is actually merged.

Reply:
🤦‍♂️
My fault, did not understood the the text right. Somehow my brain interpreted the `and` logically as an `or`